### DISCUSSION TITLE
Break URL in Github Blog article

### DISCUSSION THREAD

**Discussion by xyb** [community/community#22061] [New to GitHub]
<p>I have noticed the Github Blog article <a href="https://github.blog/2020-07-27-highlights-from-git-2-28/#changed-path-bloom-filters" rel="nofollow noopener">Highlights from Git 2.28</a> has a broken URL:</p>
<pre><code class="lang-auto">https://devblogs.microsoft.com/devops/updates-to%20the-git-commit-graph-feature/
</code></pre>
<p>which should be:</p>
<pre><code class="lang-auto">https://devblogs.microsoft.com/devops/updates-to-the-git-commit-graph-feature/
</code></pre>

**Comment by lee-dohm** [comment, marked as answer]
<p>Welcome to the community!</p>
<p>Thanks for reaching out to let us know. It appears that it’s been fixed now :bowing_man:</p>

**Comment by xyb** [comment]
<p>Hmm, but it’s still broken right now.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/172520c3553c5269d2b939da2b05197489f582ba.png" data-download-href="/uploads/short-url/3iKwKdmax05lf6xrGrXvM8QWNcu.png?dl=1" title="截屏2020-07-30下午12.21.56" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/153287/181068741-15b05a26-b2a3-4862-883f-9c3f78bcee41.png" alt="截屏2020-07-30下午12.21.56" data-base62-sha1="3iKwKdmax05lf6xrGrXvM8QWNcu" width="690" height="95" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/1/172520c3553c5269d2b939da2b05197489f582ba_2_690x95.png, https://aws1.discourse-cdn.com/github/original/2X/1/172520c3553c5269d2b939da2b05197489f582ba.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/1/172520c3553c5269d2b939da2b05197489f582ba.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/172520c3553c5269d2b939da2b05197489f582ba_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">截屏2020-07-30下午12.21.56</span><span class="informations">841×117 28 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by lee-dohm** [comment]
<p>Ah, thank you for the extra context. I missed which link you were referring to. I’ll let the team know.</p>
